src/main/java/com/revature/assignforce/validators/IsValidInterval.java

```diff
@@ -0,0 +1,49 @@
+package com.revature.assignforce.validators;
+
+
+import java.lang.annotation.ElementType;
+import static java.lang.annotation.ElementType.ANNOTATION_TYPE;
+import static java.lang.annotation.ElementType.TYPE;
+import java.lang.annotation.Repeatable;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+import java.lang.annotation.Target;
+
+import javax.validation.Constraint;
+import javax.validation.Payload;
+
+/**
+ *
```

Comment:
Add some detailed documentation comments here. It's important for people coming later to understand what this does and how it is to be used. Also use full name for @author annotations